UI/GameSettingsScreen.cpp

```diff
@@ -977,7 +977,9 @@ void GameSettingsScreen::CreateSystemSettings(UI::ViewGroup *systemSettings) {
 
 	if (System_GetPropertyBool(SYSPROP_HAS_OPEN_DIRECTORY)) {
 		systemSettings->Add(new Choice(sy->T("Show Memory Stick folder")))->OnClick.Add([](UI::EventParams &p) {
-			System_ShowFileInFolder(File::ResolvePath(g_Config.memStickDirectory.ToString()).c_str());
+			// TODO: Should build ResolvePath into System_ShowFileInFolder()?
```

Comment:
Actually, I think it is on Windows?  Where it is needed.  Probably not needed on Linux, at least.

-[Unknown]